quickstarts/router-ext/pom.xml

```diff
@@ -0,0 +1,35 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.kie.server.demo</groupId>
+    <artifactId>router-ext</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>KIE Server Router Extensions</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.kie.server</groupId>
+            <artifactId>kie-server-router-proxy</artifactId>
+            <version>7.43.0-SNAPSHOT</version>
```

Comment:
Wouldn't it have more sense to provide released version? Snapshots are not available on Maven central.

Reply:
yes, once we have the final release, I'll be updating here as well.